docs/tutorial/t-spark-monitoring.md

```diff
@@ -0,0 +1,350 @@
+## Monitoring the Spark Cluster
+
+By default, Spark stores the logs of drivers and executors as pod logs in the local file system, which gets lost if the pods are deleted. Spark provides us with the feature to store these logs in S3 so that they can later be retrieved and visualized.
+
+Monitoring of Spark cluster can be done in two ways,
+1. Using Spark History Server 
+2. Using Canonical Observability Stack (COS)
```

Comment:
I believe it makes sense to explain to the user that these two tools address two different kind of users, see [here](https://github.com/deusebio/gitex-2023-charmed-spark?tab=readme-ov-file#fetch-your-endpoints)